src/ro/siit/ObjectContainers/Main.java

```diff
@@ -0,0 +1,83 @@
+package ro.siit.ObjectContainers;
+
+import java.util.*;
+
+/**
+ * This code will store some persons in a TreeSet. After that, it will iterate through the TreeSet using 2 Comporators.
+ * For one person, it will display their hobbies using a HashMap.
+ * @author Catalina Neagu
+ */
+
+
+public class Main {
+    public static void main(String[] args) {
+        Person p1 = new Person("Ion", 10);
+        Person p2 = new Person("Andrei", 12);
+        Person p3 = new Person("Alexandra", 20);
+        Person p4 = new Person("Toma", 25);
+
+        //Iterate through the TreeSet using name comparator (String comparison)
+        TreeSet<Person> personComp = new TreeSet<Person>(new MyNameComp());
```

Comment:
It's better to have the interface as the variable's type (`Set` instead of `TreeSet`). Remember the "program against interfaces instead of implementations" mantra. :)
Same goes for other variables below.